test/recipes/70-test_key_share.t

```diff
@@ -186,13 +186,13 @@ $proxy->clear();
 $direction = SERVER_TO_CLIENT;
 $testtype = LOOK_ONLY;
 $selectedgroupid = 0;
-if (disabled("ec")) {
+if (disabled("ec") || disabled("ecx")) {
     $proxy->clientflags("-groups ffdhe3072:ffdhe2048");
 } else {
     $proxy->clientflags("-groups P-256:X25519");
 }
 $proxy->start();
-if (disabled("ec")) {
+if (disabled("ec") || disabled("ecx")) {
```

Comment:
disabled("ecx") should be sufficient

Reply:
and change X25519 to P-384 in L192 to avoid error: 
t1_lib.c:1040:group 'X25519' cannot be set